### PR TITLE
5.4.3 Gör om så att order i kitchenView baseras på den listan av menyitems som finns i order-objektet.

### PR DESCRIPTION
**Pull request by AdamRohdell**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ const server = http.createServer(app);
 const io = socketIo(server);
 const cors = require("cors");
 
+let latestOrdernr = 0;
```
Comment (steget) [inline server.js:13]: camelcasing

File: server.js [hunk 2]
```diff
@@ -70,20 +74,25 @@ app.post("/api/add_menuItem", (req, res) => {
 // Inserts a new order into the orderMap
 app.post("/api/submit_order", function (req, res) {
     var order = req.body;
+    console.log(order);
```
Comment (steget) [inline server.js:77]: log

File: server.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ const server = http.createServer(app);
 const io = socketIo(server);
 const cors = require("cors");
 
+let latestOrdernr = 0;
+let newItemId = 0;
```
Comment (steget) [inline server.js:14]: although temp, doesn't this represent latestItemId?

File: client/src/components/order.js [hunk 4]
```diff
@@ -6,25 +6,28 @@ class Order extends Component {
...
             buttonText: "",
+            totalPrice: 0,
         };
 
         this.doRightButtonAction = this.doRightButtonAction.bind(this);
     }
 
     //Method that is called immediatly after render(). This is where we make the fetch-calls
     //to the backend to fetch data to fill the order with.
+
```
Comment (steget) [inline client/src/components/order.js:22]: ?

File: client/src/components/orders.js [hunk 5]
```diff
@@ -81,9 +81,10 @@ class Orders extends Component {
...
                                     <Order
                                         orderNr={order.orderNr}
+                                        items={order.items}
                                         status={order.status}
                                         buttonText={"Redo"}
-                                        description={order.description}
+                                        totalPrice={order.totalPrice}
                                         onDelete={this.handleDelete}
                                         onItemChanged={this.fetchOrders}
                                     />
```
Comment (steget) [inline client/src/components/orders.js:90]: send name from the order as a prop aswell

File: client/src/components/orders.js [hunk 6]
```diff
@@ -81,9 +81,11 @@ class Orders extends Component {
                                 <div key={order.orderNr} className="col">
```
Comment (Minus98) [inline client/src/components/orders.js:81]: Just like with the queue display, I would sugest adding some bottom margin by adding class "mb-2" here

**Pull request merged**